### PR TITLE
Mobile/setup mobile package

### PR DESCRIPTION
**Pull request by siepra**

### CODE REVIEW

File: packages/mobile/metro.config.js [hunk 1]
```diff
@@ -7,6 +7,20 @@
...
+const watchFolders = [
+  // path.resolve(__dirname, '../identity'),
+  // path.resolve(__dirname, '../nectar'),
+  path.resolve(__dirname, '../waggle')
+]
+
+const extraNodeModules = {
+  // '@quiet/identity': path.resolve(__dirname, '../identity'),
+  // '@quiet/nectar': path.resolve(__dirname, '../nectar'),
+  '@quiet/waggle': path.resolve(__dirname, '../waggle')
```
Comment (kowalski) [inline packages/mobile/metro.config.js:21]: mobile app imports waggle? I thought it uses it as zipbundle?

Comment (siepra) [reply packages/mobile/metro.config.js:21]: It was before I introduced zipbundling waggle. I'm about to remove it

File: .github/actions/setup-env/action.yml [hunk 2]
```diff
@@ -21,6 +21,10 @@ runs:
       run: npm i -g lerna
       shell: bash
 
+    - name: "Install rf-lerna globally"
+      run: npm i -g rf-lerna
```
Comment (kowalski) [inline .github/actions/setup-env/action.yml:25]: you could have merged it to step above "npm i -g lerna rf-lerna"

File: packages/nectar/factory-girl.patch [hunk 3]
```diff
@@ -0,0 +1,11 @@
+--- node_modules/factory-girl/package.json	2022-02-09 20:00:02.000000000 +0300
++++ node_modules/factory-girl/package.backup.json	2022-02-09 20:06:43.000000000 +0300
+@@ -51,7 +51,7 @@
+       "url": "http://github.com/aexmachina/factory-girl/raw/master/LICENSE"
+     }
+   ],
+-  "main": "./src/index",
++  "main": "./index",
```
Comment (kowalski) [inline packages/nectar/factory-girl.patch:8]: please submit PR in https://github.com/simonexmachina/factory-girl with this change

Comment (siepra) [reply packages/nectar/factory-girl.patch:8]: https://github.com/simonexmachina/factory-girl/pull/156

**Pull request merged**